docs/core/install/macos.md

```diff
@@ -127,17 +127,50 @@ export PATH=$PATH:$DOTNET_ROOT
 
 This approach lets you install different versions into separate locations and choose explicitly which one to use by which application.
 
+## Arm-based Macs
```

Comment:
Is Macs correct? Or should it be macOS

Reply:
@IEvangelist  I wasn't sure, but it seems to be how its talked about:

https://appleinsider.com/inside/arm-mac